module_info_partner/models/module_information.py

```diff
@@ -1,20 +1,34 @@
-from odoo import models, fields, api
+from odoo import api, fields, models
+
+
+class HostRepository(models.Model):
+    _name = "host.repository"
+    _rec_name = "name"
+    _description = "List of all module repository"
+
+    name = fields.Char(string="Repository Name", readonly=True, index=True)
+    organization = fields.Char(string="Organization Name")
+    url = fields.Char()
+    modules_ids = fields.One2many(
+        "module.information", "host_repository_id", string="Module information"
+    )
 
 
 class ModuleInformation(models.Model):
     _name = "module.information"
-    _rec_name = 'technical_name'
+    _rec_name = "technical_name"
     _description = "Module Information and availability"
 
-    # TODO add unique constraint
     technical_name = fields.Char(string="Technical Name", readonly=True, index=True)
     name = fields.Char(readonly=True)
     description_rst = fields.Text(readonly=True)
-    description_html = fields.Html(readonly=True)
     short_description = fields.Text(
         help="Edit this field to store complementary information about the " "module"
     )
     authors = fields.Char(readonly=True)
+    host_repository_id = fields.Many2one(
```

Comment:
add the field partner_id for custom

Reply:
The partner storage for custom is done in module.partner: 
https://github.com/akretion/partner-module-information/blob/14.0-mig-partner-module-information/module_info_partner/models/module_partner.py#L8